geomstats/_backend/numpy/__init__.py

```diff
@@ -211,31 +211,10 @@ def vectorize(x, pyfunc, multiple_args=False, signature=None, **kwargs):
     return np.vectorize(pyfunc, signature=signature)(x)
 
 
-# XXX(nkoep): Can we get rid of this now?
-def cond(pred, true_fn, false_fn):
-    if pred:
-        return true_fn()
-    return false_fn()
-
-
 def cast(x, dtype):
     return x.astype(dtype)
 
 
-def diag(x):
```

Comment:
We don't need this anymore? How about the vectorization of 'diag`?

Reply:
Yes, sorry, I created a bit of a mess with the issues and PRs relating to this. @ythanwerdas pointed out that his `from_vector_to_diagonal_matrix` function does exactly what `diag` does, but in a backend-independent fashion. Since we don't seem to be using numpy's `diag` function anywhere, I removed it altogether (at least until someone actually needs it again).